pyccl/nl_pt/tracers.py

```diff
@@ -14,26 +14,27 @@
 def translate_IA_norm(cosmo, *, z, a1=1.0, a1delta=None, a2=None,
                       Om_m2_for_c2=False, Om_m_fid=0.3):
     """
-    Function to convert from a_ia values to c_ia values,
-    using the standard convention of Blazek 2019 or the variant used
-    by the Dark Energy Survey analysis.
+    Function to convert from :math:`A_{ia}` values to :math:`c_{ia}` values,
+    for the intrinsic alignment bias parameters using the standard
+    convention of `Blazek et al. 2019 <https://arxiv.org/abs/1708.09247>`_
+    or the variant used by the Dark Energy Survey analysis.
 
     Args:
         cosmo (:class:`~pyccl.core.Cosmology`): cosmology object.
-        z (float or array_like): z value(s) where amplitude is evaluated
-        a1 (float or array_like): IA a1 at input z values. Defaults to 1.0
-        a1delta (float or array_like): IA a1delta at input z values.
-            Defaults to None.
-        a2 (float or array_like): IA a2 at input z values.
-            Defaults to None.
-        Om_m2_for_c2 (bool): True to use the Blazek 2019 convention of
-            Om_m^2 scaling. Defaults to False
-        Om_m_fid (float): Value for Blazek 2019 scaling. Defaults to 0.3.
+        z (float or array_like): z value(s) where amplitude is evaluated.
+        a1 (float or array_like): IA ``a1`` at input z values.
+        a1delta (float or array_like): IA ``a1delta`` at input z values.
+        a2 (float or array_like): IA ``a2`` at input z values.
+        Om_m2_for_c2 (bool): True to use the Blazek et al. 2019 convention of
+            :math:`\\Omega_m^2` scaling.
+        Om_m_fid (float): Value for Blazek et al. 2019 scaling.
 
     Returns:
-        c1 (float or array_like): IA c1 at input z values
-        c1delta (float or array_like): IA c1delta at input z values
-        c2 (float or array_like): IA c2 at input z values
+        Tuple of IA bias parameters
+
+        - c1 (float or array_like): IA ``c1`` at input z values
+        - c1delta (float or array_like): IA ``c1delta`` at input z values
```

Comment:
Same here

Reply:
Done

Reply:
For some reason the whole line is highlighted

Reply:
Good catch! Done